dirty_cat/similarity_encoder.py

```diff
@@ -9,6 +10,46 @@
 from sklearn.utils import check_random_state
 
 from . import string_distances
+from .string_distances import get_ngram_count, preprocess
+
+
+def _ngram_similarity_one_sample(
```

Comment:
Maybe add "inplace" to the name, and clearly indicate that you are modifying inplace on variables (super important to understand the code).

Reply:
indeed!